### PR TITLE
More blindfold recipes

### PR DESCRIPTION
**Pull request by carpevenenum**
Now it can also be crafted from scarves.

### CODE REVIEW

**Comment by pisskop** [comment]
I would argue that 'long' scarves should be excluded.  Long scarves are long enough wrap around your neck and stick your hands into little pockets at the end -more than your armspan, or taller than you are-.

Too long is too long.

**Comment by shedya** [comment]
It's okay if it's longer than your armspan, I see nothing wrong with the non-wrapped parts just hanging about. If it's actually longer than the height of PC, perhaps there should be a long blindfold that also encumbers your feet by getting in the way while walking?

**Comment by Mecares** [comment]
Well one could use bandages or duct tape as well i think.

**Comment by shedya** [comment]
Bandages are soaked in disinfectant and duct tape is very sticky.

**Comment by Mecares** [comment]
Disinfectant evaporates and Duct tape stops being sticky once you fold it once with the sticky sides touching...

**Comment by shedya** [comment]
Only alcohol evaporates that fast and it's difficult to fold the tape precisely side-to-side.

**Comment by Barhandar** [comment]
It is, however, perfectly possible to fold the remaining sticky parts onto themselves.

CDDA bandages are, according to recipe, dressing+gauze combo, so they're perfectly usable as blindfold if you remove the dressing. Or let the disinfectant evaporate, because majority of it in the recipes is alcohols anyway.

**Comment by shedya** [comment]
Mmmmhhmmmmm i'm convinced.

File: data/json/recipes/recipe_armor.json [hunk 1]
```diff
@@ -2424,7 +2424,41 @@
...
+      [ "scarf_fur_loose", 1 ],
+      [ "scarf_fur_long_loose", 1 ],
+      [ "knit_scarf_loose", 1 ],
+      [ "long_knit_scarf_loose", 1 ],
+      [ "bandages", 2 ]
+    ]
+  ]
+},
+{
+  "type" : "recipe",
```
Comment (Rivet-the-Zombie) [inline data/json/recipes/recipe_armor.json]: Why does this need a separate recipe?

Comment (shedya) [inline data/json/recipes/recipe_armor.json]: To use fabrication instead of tailoring

**Comment by fubeca** [comment]
This new recipe causes a naming collision as below:

screen shot 2016-09-01 at 07 00 04

**Comment by Rivet-the-Zombie** [comment]
A [fix](https://github.com/CleverRaven/Cataclysm-DDA/pull/18178) has been merged.

**Pull request merged**